crates/language-tests/tests/language/reference/array.surql

```diff
@@ -1,74 +0,0 @@
-/**
-[test]
-wip = true
-
-[[test.results]]
-value = "NONE"
-
-[[test.results]]
-value = "NONE"
-
-[[test.results]]
-value = "[{ id: house:one, using: [] }]"
-
-[[test.results]]
-value = "[{ id: utility:gas }, { id: utility:water }]"
-
-[[test.results]]
-value = "[{ connected_to: [house:one], id: utility:gas }]"
-
-[[test.results]]
-value = "[{ connected_to: [house:one], id: utility:water }]"
-
-[[test.results]]
-value = "{ id: house:one, using: [utility:gas, utility:water] }"
-
-[[test.results]]
-error = "Cannot delete `house:one` as it is referenced by `utility:gas` with an ON DELETE REJECT clause"
-
-[[test.results]]
-value = "[{ connected_to: [], id: utility:gas }]"
-
-[[test.results]]
-value = "[{ connected_to: [], id: utility:water }]"
-
-[[test.results]]
-value = "[{ connected_to: [], id: utility:gas }, { connected_to: [], id: utility:water }]"
-
-[[test.results]]
-value = "[{ id: house:one, using: [] }]"
-
-[[test.results]]
-value = "[]"
-
-[[test.results]]
-value = "[]"
-
-[[test.results]]
-value = "NONE"
-
-[env.capabilities]
-allow-experimental = ["record_references"]
-
-*/
-DEFINE FIELD connected_to ON utility TYPE option<array<record<house>>> REFERENCE ON DELETE REJECT;
-DEFINE FIELD using ON house TYPE references<utility>;
```

Comment:
Only this has become invalid, but should be replaced with the new syntax when it comes. Can we mark the tests as wip instead of delete them?